src/css/controls/flags/tizen.less

```diff
@@ -0,0 +1,222 @@
+@import "../../shared-imports/vars.less";
+
+.jw-tizen-app {
+    // Increasing the size of the svg-icons - icon.less
+    .jw-svg-icon {
+        .square(60px);
```

Comment:
no tizen-icon size ?

Reply:
I didn't think it was necessary for this since this styling is more of a placeholder. The actual app's styling will most likely be quite different, so I just threw this one together.